src/browser/linkedom.js

```diff
@@ -1 +1,2 @@
 export const DOMParser = global.DOMParser
+export const parseHTML = global.parseHTML
```

Comment:
There isn't a `parseHTML` in browser.You need provide that or use `DOMParser`

Reply:
My bad. It was a conflict with my project